go/heif/heif.go

```diff
@@ -608,12 +649,14 @@ type ImageAccess struct {
 
 func (img *Image) GetPlane(channel Channel) (*ImageAccess, error) {
 	height := C.heif_image_get_height(img.image, uint32(channel))
+	runtime.KeepAlive(img) // TODO: Is this really neccessary? Will be called again in the same func.
 	if height == -1 {
 		return nil, fmt.Errorf("No such channel")
 	}
 
 	var stride C.int
 	plane := C.heif_image_get_plane(img.image, uint32(channel), &stride)
+	runtime.KeepAlive(img)
```

Comment:
You can remove this, too. `img` is still used below and stored in the newly created `ImageAccess` object.

Reply:
See above for why it will stay.